lib/widgets/action_sheet.dart

```diff
@@ -60,16 +83,65 @@ abstract class MessageActionSheetMenuItemButton extends StatelessWidget {
   String label(ZulipLocalizations zulipLocalizations);
   void Function(BuildContext) get onPressed;
 
+  late final bool _isFirst;
+  late final bool _isLast;
+
+  void setIsFirst(){
+      _isFirst = true;
+  }
```

Comment:
This mutation of a widget object is awfully unidiomatic for Flutter. We'll want to find a solution which doesn't involve mutating any widgets after constructing them.

It's possible that a good solution here will involve making more widgets, or different widgets — the `MessageActionSheetMenuItemButton` class as a subclass of `StatelessWidget` is something we made up, and we're free to change it if that helps us structure things well. If you're trying ideas for this and you have questions about whether a particular refactor seems like a good direction, `#mobile-team` on chat.zulip.org will be a good place to ask.

Reply:
Ok Sure, I will try to find some way around to avoid that behaviour. Let's discuss more about that on `#mobile-team` on chat.zulip.org